diff --git a/dist/cadre-node.js b/dist/cadre-node.js
index 4240c35d9e7dd4f497677ae0a4270bcf64936d7e..951f9457305a310f8ad9c7b663b1aef84fb32457 100644
--- a/dist/cadre-node.js
+++ b/dist/cadre-node.js
@@ -207,7 +207,8 @@ export class CadreNode {
             arachnode: { enableRingZulu: this.config.profile === 'storage' },
             ...(privateKey && { privateKey }),
             ...(network?.transports && { transports: network.transports }),
-            ...(network?.listenAddrs && { listenAddrs: network.listenAddrs })
+            ...(network?.listenAddrs && { listenAddrs: network.listenAddrs }),
+            ...(network?.connectionGater && { connectionGater: network.connectionGater })
         };
         return await createLibp2pNode(nodeOptions);
     }
